react.md

```diff
@@ -4,30 +4,28 @@ Try your best to answer each question on your own before looking up the answer o
 
 #### 1. Here is a list of pros and cons to using the React library to build your application -- but some of them are false. Remove the false statements from the list:
 
-- React was created to be simple, so that even people with minimal code experience could use it and create Single Page Applications (SPAs)
 - React is a modern, efficient answer to complex UI applications
-- React is a full stack framework for modern web applications
-- React is a flexible library that plays the role of V in an MVC framework
+- React is a full stack framework for modern web application- React is a flexible library that plays the role of V in an MVC framework
```

Comment:
Close! React is not quite a full stack framework. The key here is that React only allows us to create what the user sees as part of the V in MVC. The database, controllers, and models must be handled differently. But we will dive in more deeply on that later on!